types/block.go

```diff
@@ -56,42 +49,12 @@ type IntermediateStateRoots struct {
 	RawRootsList [][]byte
 }
 
-// ToABCICommit converts Rollkit commit into commit format defined by ABCI.
-// This function only converts fields that are available in Rollkit commit.
-// Other fields (especially ValidatorAddress and Timestamp of Signature) has to be filled by caller.
-func (c *Commit) ToABCICommit(height uint64, hash Hash, val cmtypes.Address, time time.Time) *cmtypes.Commit {
```

Comment:
not sure if changing `ToABCICommit` to `GetABCICommit` was needed? you could leave it as is? 
`func (s *Signature) ToABCICommit ...`, unless `GetABCICommit` is needed somewhere else?

Reply:
Refer to https://github.com/rollkit/rollkit/pull/1739#discussion_r1664627348 for context